jobbergate-slurp/slurp/migrators/job_scripts.py

```diff
@@ -17,7 +17,7 @@ def migrate_job_scripts(nextgen_db, legacy_job_scripts, user_map, application_ma
     job_scripts_map = {}
     logger.info("Migrating job_scripts to nextgen database")
 
-    s3man.nextgen.engine.ensure_bucket()
+    legacy_id_empty_jobscript = set()
```

Comment:
This name is a bit confusing. Maybe `empty_legacy_jobscript_ids`?

Reply:
Removed on the speed up refactoring https://github.com/omnivector-solutions/jobbergate/pull/181